README.md

```diff
@@ -40,25 +40,29 @@ See the License for the specific language governing permissions and
 limitations under the License.
 
 
+[maven]: https://maven-badges.herokuapp.com/maven-central/com.snowplowanalytics/schema-ddl_2.12
+[maven-image]: https://maven-badges.herokuapp.com/maven-central/com.snowplowanalytics/schema-ddl_2.12/badge.svg
+
 [license-image]: http://img.shields.io/badge/license-Apache--2-blue.svg?style=flat
 [license]: http://www.apache.org/licenses/LICENSE-2.0
 
-[maven-badge]: https://maven-badges.herokuapp.com/maven-central/com.snowplowanalytics/schema-ddl_2.12/badge.svg
-[maven-link]: https://maven-badges.herokuapp.com/maven-central/com.snowplowanalytics/schema-ddl_2.12
+[build]: https://github.com/snowplow-incubator/schema-ddl/actions?query=workflow%3A%22Test+and+deploy%22
+[build-image]: https://github.com/snowplow-incubator/schema-ddl/workflows/Test%20and%20deploy/badge.svg
 
-[travis]: https://travis-ci.org/snowplow-incubator/schema-ddl
-[travis-image]: https://travis-ci.org/snowplow-incubator/schema-ddl.png?branch=master
+[coveralls]: https://coveralls.io/github/snowplow-incubator/schema-ddl?branch=master
+[coveralls-image]: https://coveralls.io/repos/github/snowplow-incubator/schema-ddl/badge.svg?branch=master
 
 [snowplow]: https://github.com/snowplow/snowplow
-[iglu]: https://github.com/snowplow/iglu
 [self-describing]: http://snowplowanalytics.com/blog/2014/05/15/introducing-self-describing-jsons/
 
-[techdocs]: https://github.com/snowplow/iglu/wiki/
-[roadmap]: https://github.com/snowplow/iglu/wiki/Product-roadmap
-[setup]: https://github.com/snowplow/iglu/wiki/
-[contributing]: https://github.com/snowplow/iglu/wiki/Contributing
+[developer-guide]: https://github.com/snowplow-incubator/schema-ddl/wiki/
```

Comment:
Maybe that's time to move to docs ?

Reply:
That's a good point - we had a talk with @ebedford420 recently and decided it would make sense to keep very low-level developer-only documentation out of docs website to not pollute it with details that probably nobody except few engineers inside Snowplow engineering team will need.

I think those "very low-level developer-only" libraries are currently only schema-ddl and scala-iglu-client.

Reply:
Makes sense !